tests/integration/parallel_skip.yaml

```diff
@@ -2,6 +2,7 @@
 # The list of "directories/", "directory/file.py", or "directory_prefix" (acting as "directory_prefix*" glob)
 # tests that will be launched sequentially without other parallel running tests
 # After adding something else, sort it for future yourself 💕
+- test_aligned_cache_s3/
```

Comment:
What is the problem with running it in parallel? It will not run tests inside test_aligned_cache_s3 in parallel to each other, but just in parallel to other tests and for each test suit a different clickhouse server is anyway used.